### ISSUE TITLE
Create Teacher UI

### ISSUE THREAD

**Issue by AllanJeremy** [AllanJeremy/esomo-upgrade#3]
Started designing the teacher UI, got tired since was coding the whole day. Feel free to rearrange tab. Dashboard can be copy pasted from the Superuser

Here's [the design](https://drive.google.com/file/d/0B0kPYrW8iZenc3pyOVcwR3ZFcTA/view?usp=sharing) so far. As you do that I'll finish up on superuser if I can. Then do more designs for the other parts

**Comment by mwauramuchiri** [comment]
What do you think we rename dashboard to 'my account' for the superuser and teacher because it's about changing their passwords and username.
For the principle (principal*), my idea on the dashboard tab was having brief info on almost everything... assignments, classrooms etc

**Comment by mwauramuchiri** [comment]
Also, I think we should redesign where we use the tabs except for the superuser.
Most have a create tab and a manage tab.
I think there should not be the create and manage tabs for teachers. Their **primary work** is to interact with the students and teaching not creating classrooms/... and also it will reach a point there will be no more classrooms to be created, right? So instead of kujazisha the file with all these create forms and all, I'm proposing to have just a button at the right side of the tab title then since most of these actions will be done using ajax to make the site seem like google classrooms, I create a js template file where all those forms will be there then when the teacher clicks on, say 'add classroom', either a modal or a right side-nav appears with the right forms. On submit, the new classroom just appears in the list without the page loading.
This is also for the edit classroom functionality and others instead of kujaza the file with different hidden modal...

In short, let the teacher do almost everything with the least number of navigations.
So it will be like js classes with different functions, almost like the php classes  file.
What do you think?
Lights just went off.
This comment was to be sent by 6:30am and I can't retype all these to sms

**Comment by AllanJeremy** [comment]
> Also, I think we should redesign where we use the tabs except for the superuser.
> Most have a create tab and a manage tab.
> I think there should not be the create and manage tabs for teachers. Their primary work is to interact with the students and teaching not creating classrooms/... and also it will reach a point there will be no more classrooms to be created, right? So instead of kujazisha the file with all these create forms and all, I'm proposing to have just a button at the right side of the tab title then since most of these actions will be done using ajax to make the site seem like google classrooms, I create a js template file where all those forms will be there then when the teacher clicks on, say 'add classroom', either a modal or a right side-nav appears with the right forms. On submit, the new classroom just appears in the list without the page loading.
> This is also for the edit classroom functionality and others instead of kujaza the file with different hidden modal...
> 
> In short, let the teacher do almost everything with the least number of navigations.
> So it will be like js classes with different functions, almost like the php classes file.
> What do you think?


Good idea, looking forward to seeing how you intend to pull it off. 1 day behind schedule with superuser but trying to get it finished by end of today. Can we maybe meet on Friday and google hangout before that, so you can help me understand how it's gonna look, I checked out the google classroom video more keenly, trying to understand how we'd pull it off though. 

> Lights just went off.
> This comment was to be sent by 6:30am and I can't retype all these to sms

 It's fine, just logged onto github anyway
